llvm/lib/Transforms/IPO/MemProfContextDisambiguation.cpp

```diff
@@ -2146,6 +2204,21 @@ bool ModuleCallsiteContextGraph::calleeMatchesFunc(
   return true;
 }
 
+bool ModuleCallsiteContextGraph::sameCallee(Instruction *Call1,
+                                            Instruction *Call2) {
+  auto *CB1 = dyn_cast<CallBase>(Call1);
```

Comment:
Do we need to check the dyn_cast returned values for nulls? If not then should they be dyn_cast in the first place?

Reply:
changed to cast here.